Code/GraphMol/DistGeomHelpers/testDgeomHelpers.cpp

```diff
@@ -550,6 +550,28 @@ void testMultipleConfs() {
 
 }
 
+void testMultipleConfsExpTors() {
+  std::string smi = "CC(C)(C)c(cc1)ccc1c(cc23)n[n]3C(=O)/C(=C\\N2)C(=O)OCC";
+  ROMol *m = SmilesToMol(smi, 0, 1);
+  INT_VECT cids = DGeomHelpers::EmbedMultipleConfs(*m, 10, 30, 100, true,
+               false,-1, true, 1, -1.0, 0, 1e-3, false, true, true, true, false);
+  INT_VECT_CI ci;
+  SDWriter writer("junk.sdf");
```

Comment:
This SDWriter should probably be commented out for the release version. I think it was just there for testing.